docs/tutorials/drm-config.md

```diff
@@ -145,7 +145,7 @@ playback.  Passing in a higher security level than can be supported will cause
 default is the empty string, which is the lowest security level supported by the
 key system.
 
-Each key system has their own values for robustness.  The values for Widevine
+Each key system has its values for robustness.  The values for Widevine
```

Comment:
I don't quite agree with this particular change. "their own" better conveyed the idea the key systems have DIFFERENT sets of robustness values. Perhaps "its own"?

Reply:
When inanimate objects are kept in perspective, we use *its* instead of *their*.

Reply:
Hello @CodeDotJS , thank you so much for your interest in contributing to Shaka Player, and thank you for reading our docs carefully! 

The docs are meant to help our developers understand how Shaka Player works easily and quickly. More than half of our team members are native English speakers, and we had checked very carefully before we published the docs, to make sure the docs are fairly easy to read and understand. At this point with our limited capacity and core projects priorities, it may not be the best use of both our time and energies to double check, explain and arbitrate the spelling/grammar, especially with a few rounds of reviews and updates.

Our original goal to volunteer at GSoC is to help students to gain more tech and coding experience through our project, and potentially become long-term contributors of Shaka Player. We want to invest our time to help the external developer to understand our code base, and finish projects in our code base. So instead of focusing on the grammar in the docs, I encourage you to look into the projects list and maybe enhancements list if you are interested to know more about thee code base. 

If you're cool with that, please take your time and update the PR one final round by addressing @theodab 's comments. Thank you for your understanding!